build/phase.go

```diff
@@ -161,6 +161,15 @@ func (p *Phase) Run(ctx context.Context) error {
 		return errors.Wrapf(err, "failed to copy files to '%s' container", p.name)
 	}
 
+	p.logger.Debugf("Running container: %v", p.ctr.ID)
```

Comment:
I don't see the value in providing the container ID w/o providing a way to prevent the intermediate containers from being removed after running.

Reply:
I think making the cmd the top-level output (i.e. the output without the prepended spaces), rather than the container ID, would be a good alternative, as the cmd describes which phase is being run.